### PR TITLE
tools: support linux output_root when claiming output_base

### PR DESCRIPTION
**Pull request by dt**
Release note: none.

Epic: none.

### CODE REVIEW

**Review by rail — APPROVED** [review]

File: tools/claim_output_base.sh [hunk 1]
```diff
@@ -20,27 +20,30 @@ if [ ! -z "${BAZEL_OUTPUT_BASE-}" ]; then
   return
 fi
 
+if [ -d "${HOME}/.cache/bazel/_bazel_${USER}" ]; then
+  SUFFIX="$(pwd | md5sum | head -c6)"
+  OUTPUT_ROOT="${HOME}/.cache/bazel/_bazel_${USER}"
+else
+  SUFFIX="$(pwd | md5 | head -c6)"
+  OUTPUT_ROOT="/private/var/tmp/_bazel_${USER}"
```
Comment (knz) [inline tools/claim_output_base.sh:28]: why not just `/var/tmp/...` in this case?

**Review by knz — APPROVED** [review]
thank you!

File: tools/claim_output_base.sh (hunk 1 above)
Comment (dt) [reply tools/claim_output_base.sh:28]: `/private/var/tmp` is what is in the bazel docs and what `bazel info output_root` returns. I don't know how they come up with it (the code is `/var` but presumably they readlink or something somewhere) but I also don't really want to know -- I just want to take what they _are_ using and append something.

Comment (dt) [reply tools/claim_output_base.sh:28]: does that work for you?

Comment (knz) [reply tools/claim_output_base.sh:28]: i had no objection the first time around. i have already approved.

**Comment by dt** [comment]
TFTRs!

bors r+

**Pull request merged**